requirements.txt

```diff
@@ -1 +1,4 @@
 pytest==8.1.1
+fastapi==0.68.1
+uvicorn==0.15.0
+requests==2.26.0
```

Comment:
Why do we need requests?

Reply:
The `requests` module is required by the `starlette.testclient` module, which is a dependency of the `fastapi.testclient` module that we use for testing our FastAPI application. Without `requests`, our tests would fail with an ImportError.